xml/Microsoft.Build.Globbing/MSBuildGlobWithGaps.xml

```diff
@@ -19,8 +19,17 @@
   </Interfaces>
   <Docs>
     <summary>
-    A glob with gaps. The gaps are represented as other globs.
-    </summary>
+                A glob with gaps. The gaps are represented as other globs.
```

Comment:
This section is messed up: https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.build.globbing.msbuildglobwithgaps?view=msbuild-15&branch=pr-en-us-4939.

Reply:
```suggestion
A glob with gaps. The gaps are represented as other globs.
```